### ISSUE TITLE
Package for Conda

### ISSUE THREAD

**Issue by medley56** [lasp/space_packet_parser#49]
Currently we only distribute via PyPI but people love Conda. Set up a Conda-amenable distribution process and document it.

**Comment by greglucas** [comment]
This seems like it would be a decent amount of effort to support for little gain IMO because this package doesn't contain any binary compilations or architecture specific wheels where conda really adds a benefit.

I guess the benefit is to be able to `conda install space_packet_parser` rather than `pip install space_packet_parser`?

https://conda-forge.org/docs/maintainer/adding_pkgs/

**Comment by medley56** [comment]
Yeah, it is a bit of a pain but there are enough people out there using conda that I want to support them. For example if someone is just running some scratch analysis stuff, I want them to be able to add space-packet-parser to their conda environment without having to manually pip install (which sort of defeats the point of conda).
